qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -98,7 +98,7 @@
             numa_memdev_shm0 = mem-mem1
             numa_nodeid_shm0 = 0
             fs_binary_extra_options = " -o cache=auto"
-            test_drivers = 'viorng viostor vioscsi viofs balloon'
+            test_drivers = 'balloon viostor vioscsi viorng viofs balloon'
```

Comment:
Hi @xiagao 

Just have a confuse, why we need 2 balloon in the test_drivers? 
Would you please help explain it more?Thanks!

Reply:
Yes, it is. 
Updated, pls help check it again, thx.